src/components/ScoreBoard/ScoreBoard.vue

```diff
@@ -0,0 +1,275 @@
+<template>
+  <div id="ScoreBoard">
+    <!-- Players col -->
+    <div id="playerNames">
+      <div id="curentTurnNumber">{{ curentTurn }}</div>
+      <div
+        v-for="(player, i) in game.getPlayers()"
+        :key="i"
+        :class="'player ' + (player.getName() == curentPlayer ? 'curent' : '')"
+        :title="player.getName()"
+      >
+        <div class="turnNumber">
+          {{ i + 1 }}
+        </div>
+        <div class="name">
+          <span>
+            {{ player.getName() }}
+          </span>
+        </div>
+      </div>
+    </div>
+
+    <!-- Score col -->
+    <div id="scores">
+      <div id="title">
+        <div class="turnNumber" v-for="turn in game.getTurn()" :key="turn">
+          {{ turn }}
+        </div>
+      </div>
+      <div id="players">
+        <div class="player" v-for="(player, i) in game.getPlayers()" :key="i">
+          <Score
+            class="score"
+            v-for="(turn, j) in game.getTurn()"
+            :key="turn"
+            :last="j == game.getTurn() - 1"
+            :score="globalScore[player.getName()][j]"
+          />
+        </div>
+      </div>
+    </div>
+
+    <!-- Total score col -->
+    <div id="totalScores">
+      <div id="title"><b>TOT</b></div>
+      <div class="score" v-for="(player, i) in game.getPlayers()" :key="i">
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Score from './Score'
+
+export default {
+  components: { Score },
+  name: 'ScoreBoard',
+  props: {
+    game: { type: Object, requiered: true },
+    curentPlayer: { type: String, default: null },
+    curentTurn: { type: Number, default: null }
```

Comment:
Vu que c'est quelque chose qui change en cours de partie autant le mettre dans data et tu exposes un setter

Reply:
Une props est aussi dynamique qu'une variable dans data

Reply:
```
[Vue warn]: Avoid mutating a prop directly since the value will be overwritten whenever the parent component re-renders. Instead, use a data or computed property based on the prop's value. Prop being mutated: "$propName"
```

Erreur que Vue va trigger quand tu vas faire donc pas conseillé de faire muté une propriété même si c'est possible